.github/workflows/unit.yml

```diff
@@ -1,5 +1,12 @@
-on: [push, pull_request]
+on:
+  push:
+    branches:  [master]
+    tags: [v*]
```

Comment:
Not a big fan of not running CI on branches. It's useful to detect failures when working on a long feature branch.

Reply:
Conclusion from a sync conversation with @galargh: This actually makes sense. When working on a long feature branch, one can just open a draft PR to trigger CI builds.